src/FamilyHubs.ReferralApi.Api/Queries/GetReferrals/GetReferralsByReferrerCommand.cs

```diff
@@ -42,7 +47,15 @@ public async Task<PaginatedList<ReferralDto>> Handle(GetReferralsByReferrerComma
             throw new NotFoundException(nameof(Referral), request.Referrer);
         }
 
-        var filteredReferrals = await entities.Select(x => new ReferralDto(
+        if (request != null && !string.IsNullOrEmpty(request.SearchText))
+        {
+            entities = entities.Where(x => x.RequestNumber.ToString().Contains(request.SearchText) ||
+                                           (x.DateRecieved != null && x.DateRecieved.Value.ToString("dd MMMM yyyy").Contains(request.SearchText)) ||
```

Comment:
I dont know enough about this and if this is correct business requirement that so be it and you can close this comment but I am not sure if its good idea to compare dates as string there are very high chances that users could make spelling mistake in entering text data time? my point is should we be displaying date time picker and  and capturing real dates and in here compare two dates?

Reply:
The user never adds any dates these are ALL taken care of by the system, also it is displaying dates as 16th March 2023, so if they put a search in it needs to find it